src/Microsoft.Azure.SignalR.Management/HubInstanceFactories/PersistentInitializationTask.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    internal class PersistentInitializationTask : IInitializationTask
+    {
+        private readonly IServiceConnectionContainer _connectionContainer;
+
+        public PersistentInitializationTask(IServiceConnectionContainer connectionContainer)
+        {
+            _connectionContainer = connectionContainer;
+        }
+
+        public async Task StartAsync(CancellationToken token)
+        {
+            _ = _connectionContainer.StartAsync();
+            await _connectionContainer.ConnectionInitializedTask.OrTimeout(token);
```

Comment:
what if Timeout? What will user see?

Reply:
After "Improve timeout error message" commit, user will get a `TimeoutException` with error message "Timeout occurred for establishing service connections after {timeout}."